nameable.rb

```diff
@@ -0,0 +1,30 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Not Implmented'
+  end
+end
+
+class Decorator < Nameable
+  attr_accessor :name
+
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecoration < Decorator
+  def correct_name
+    @nameable.correct_name
```

Comment:
- [ ] @tarikbouari kindly ensure your `correct_name` method capitalizes the first letter of the given name.

Reply:
> * [ ]  @tarikbouari kindly ensure your `correct_name` method capitalizes the first letter of the given name.

Thanks for the suggestion but the correct_name method isn't capitalized in the exercise requirement.